src/dev/build/tasks/bin/scripts/opensearch-dashboards

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 SCRIPT=$0
 
+
+
```

Comment:
Minor: were these spaces added on purpose?

Reply:
nope
It's just a leftover from previous OS check.